<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QDialog" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <widget class="QLineEdit" name="searchTxt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter the genre.</string>
   </property>
  </widget>
  <widget class="QListWidget" name="genreBookList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>581</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SearchBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QListWidget" name="favoriteList">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Do you like the descrition of this book? (Input yes if you like it)</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLbl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>571</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="yesBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>YES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeButton">
   <property name="geometry">
    <rect>
     <x>544</x>
     <y>390</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>I read this</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
